src/Microsoft.Identity.Client/ApiConfig/Parameters/AcquireTokenCommonParameters.cs

```diff
@@ -14,11 +14,11 @@ internal class AcquireTokenCommonParameters
 
         public ApiEvent.ApiIds ApiId { get; set; } = ApiEvent.ApiIds.None;
         public Guid TelemetryCorrelationId { get; set; }
+        public Guid UserProvidedTelemetryCorrelationId { get; set; }
```

Comment:
>UserProvidedTelemetryCorrelationId [](start = 20, length = 34)

This is the new one, seems like you have now started taking the param above and using that in a number of places. is that intentional?

Reply:
yes. This is used to set the correlation id that will be used by the authentication request.

Reply:
hmm, that is not what the user passed in. that seems odd to me

Reply:
I dont think so but perhaps I am mistaken? I checked the code and the TelemetryCorrelationId is the same as the CorrelationId. It seems like we have two different names for the same value in the code.